source/extensions/filters/http/cache/config.cc

```diff
@@ -0,0 +1,37 @@
+#include "extensions/filters/http/cache/config.h"
+
+#include <memory>
+
+#include "envoy/common/time.h"
+#include "envoy/config/filter/http/cache/v2alpha/cache.pb.validate.h"
+#include "envoy/registry/registry.h"
+#include "envoy/stats/scope.h"
+
+#include "extensions/filters/http/cache/cache_filter.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+
+using Http::FilterChainFactoryCallbacks;
+using Server::Configuration::FactoryContext;
+using Server::Configuration::NamedHttpFilterConfigFactory;
+using std::make_shared;
+using std::string;
```

Comment:
Let's not use these `using` statements. We have 100 chars width to play with in this codebase so it's not too bad to fully qualify.

Reply:
Done. 100 chars is certainly better than the 80 I've been dealing with, but doesn't change how many characters and tokens we have to wade through when reading code. Regardless, if Envoy style likes more, I will cheerfully comply.